src/algoliasearch.helper.js

```diff
@@ -1,479 +1,483 @@
-(function($) {
-  var extend = function(out) {
-    out = out || {};
-    for (var i = 1; i < arguments.length; i++) {
-      if (!arguments[i]) {
-        continue;
-      }
-      for (var key in arguments[i]) {
-        if (arguments[i].hasOwnProperty(key)) {
-          out[key] = arguments[i][key];
-        }
+module.exports = helper;
```

Comment:
nothing changed in this helper implementation BUT:
- had to remove the wrapping function, no more needed in commonJS/browserify world